web/controllers/v1/bundles_controller.ex

```diff
@@ -42,7 +42,7 @@ defmodule Cog.V1.BundlesController do
   def create(conn, %{"bundle" => params}) do
     case install_bundle(params) do
       {:ok, bundle} ->
-        bundle = Repo.preload(bundle, commands: [rules: [permissions: :namespace]])
+        bundle = Repo.preload(bundle, [commands: [rules: [permissions: :namespace]], namespace: [permissions: :namespace]])
```

Comment:
It's really a bummer that we have to define both sides of the `namespace` relation for `permissions`. I think that was fixed in Ecto 2.0 but we'll have to wait a bit.